ui/src/app/components/inputs.tsx

```diff
@@ -220,15 +220,15 @@ export const FormValidationErrorMessage = withStyle({
   fontSize: 12
 })('div');
 
-export const TextInput = React.forwardRef(({style = {}, onChange, onBlur, ...props}:
-      {style?: React.CSSProperties, onChange: Function, onBlur: Function, [key: string]: any},
+export const TextInput = React.forwardRef(({style = {}, type = 'text', onChange, onBlur, ...props}:
```

Comment:
Should textInput be used for anything else except type text

Reply:
Good point. I refactored to use a regular number input instead of TextInput.